### PR TITLE
Fix composer binary permissions to allow updates

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Comment by gary-kim** [comment]
Oh, I was wondering why the CI was failing.

**Comment by ChristophWurst** [comment]
It's actually because the action pulls composer v1.10.0-RC. And everyone is running the stable 1.9.3 locally. Reported it at https://github.com/shivammathur/setup-php/issues/192

**Comment by ChristophWurst** [comment]
Nothing to do here for now. Guess we just have to check the diff manually. All that is different in the upcoming composer is that it adds a comment to the generated php files.

File: .github/workflows/composer.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
         php-version: 7.4
         coverage: none
     - name: Update composer
-      run: composer self-update && composer --version
+      run: sudo composer self-update && composer --version
```
Comment (nickvergessen) [inline .github/workflows/composer.yml]:
```
$ sudo php composer help
Do not run Composer as root/super user! See https://getcomposer.org/root for details
…
```

:thinking:

Comment (ChristophWurst) [reply .github/workflows/composer.yml]: I know :) But right now it's the only solution I could find for this problem: https://github.com/shivammathur/setup-php/issues/192#issuecomment-590846575

**Pull request merged**